core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/messaginglayer/DataOutputPort.scala

```diff
@@ -17,7 +18,8 @@ import scala.collection.mutable
   * The internal logic can send data messages to other actor without knowing
   * where the actor is and without determining the sequence number.
   */
-class DataOutputPort(selfID: ActorVirtualIdentity, networkSenderActor: NetworkSenderActorRef) {
+class DataOutputPort(selfID: ActorVirtualIdentity, networkSenderActor: NetworkSenderActorRef)
+    extends LazyLogging {
```

Comment:
Is this change a part of the logging change but was put here by mistake?

Reply:
I was doing some logging here but I removed the log lines I think.